learn/app-development/deployment/app-integration-with-aws-cdn.md

```diff
@@ -0,0 +1,98 @@
+
+## Create an Amazon CloudFront distribution
+
+- Sign in to the AWS Management Console and open the [CloudFront console](https://console.aws.amazon.com/cloudfront/.), select Create Distribution
+- On the first page of the Create Distribution Wizard, in the `Web` section, choose Get Started.
+- At Origin Domain Name select previously created S3 Bucket, if you want to store content in any specific directory provide an origin path.
+- select restrict buckets access as `yes` and select previously created existing Identity.
+- At Grant Read Permissions on Bucket select `yes` for update bucket policy ,it will provide read permission to the origin access identity when you create the distribution.
+[![CDN origin settings](/learn/assets/wme-setup/cdn-origin-setting.png)](/learn/assets/wme-setup/cdn-origin-setting.png)
+
+- Provide Cache behaviour configuration details
+[![CDN cache behaviour](/learn/assets/wme-setup/cdn-cache-behaviour-settings.png)](/learn/assets/wme-setup/cdn-cache-behaviour-settings.png)
+
+- At distribution settings select Default CloudFront certificate or import custom SSL certificate , then click on create distribution by providing required values for creation.
+[![CDN distribution settings](/learn/assets/wme-setup/cdn-distribution-setting.png)](/learn/assets/wme-setup/cdn-distribution-setting.png)
+
+- Wait for few minutes for create distribution and after creating distribution note down the domain name of CloudFront distribution.
+
+## WaveMaker App Build Process for CDN support
+
+- WaveMaker app is consiting of frontend artifacts (html,css,js,images etc) and backend artififacts (Java Classes). The following approach helps you in generating separate artifacts for a WaveMaker application. The frontend artifact (static content) can be uploaded to Cloudfront for CDN and backend can be deployed to any web server.
+- To generate two different artifacts from WaveMaker application use below command. This command takes CDN_URL as input. Configure your CDN before executing this command.
+- Refer [WaveMaker app build with maven](/learn/app-development/deployment/building-with-maven) for more details on WaveMaker app building.
+
+```shell
+mvn clean install -P<profile-name> -Dcdn-url=<CDN_URL>
+```
+
+```shell
+mvn clean install -Pdeployment -Dcdn-url=https://mydomain.cloudfront.net/my_app>/1234/
+```
+
+- The above command will generate a new folder called **target** in the project sources & generates `project.war` file and `ui-artifacts.zip` file in it. The `ui-artifacts.zip` file have static files of the application, You need to unzip the file `ui-artifacts.zip` and upload it to Amazon S3 bucket that we created in the previous steps. Use the following commands for unzip and upload to S3.
+
+- For unzip the file to a specific folder
```

Comment:
Wrong. Replace with:

To unzip the file and store contents in a specific folder